RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.h

```diff
@@ -51,6 +52,7 @@ class dso_hidden MeasurementTrackerEventProducer final : public edm::stream::EDP
   bool selfUpdateSkipClusters_;
   bool switchOffPixelsIfEmpty_;
   bool isPhase2;
+  bool useVectorHits;
```

Comment:
member variables should end in underscore

Reply:
Done (also for `isPhase2`).